src/js/06-page-rating.js

```diff
@@ -74,6 +74,11 @@
         showCollectorDialog()
       })
     },
+    'fieldValues': {
+      'summary': `Feedback on ${document.title}`,
+      'customfield_11580': window.location.href,
+      'description': `\n\nBrowser environment: ${navigator.userAgent}`,
```

Comment:
NB for other reviewers: These are the defaults that are injected into the Summary, Page URL, and description fields.
We have a lot of leeway to update this with anything that is available in the Web Browser model.